src/utils/pools.spec.ts

```diff
@@ -7,8 +7,11 @@ import { getPoolReserves } from './pools';
 import { getVault } from '../constants/vaults';
 import { VaultName } from '../enums/vaultName';
 import { MainnetPoolNumbers } from '../enums/poolNumber';
+import * as dotenv from "dotenv";
 import { connectToProvider } from '../crypto/providers';
 
+dotenv.config();
```

Comment:
Remove dotenv completely actually.  I don't want the ichi-sdk to assume it exists as it is used primarily with other projects.  the code can expect process.env.INFURA_ID to exist without relying on dotenv.  And I'm not saying that other projects may not use dotenv that use the ichi-sdk, I just don't want that to be a requirement within the ichi-sdk for simplicity reasons